BridgeAppSDK/SBAProfileItem.swift

```diff
@@ -408,3 +408,28 @@ open class SBAUserDefaultsProfileItem: SBAProfileItemBase {
         return retVal
     }
 }
+
+open class SBAUserProfileItem: SBAProfileItemBase {
+    open static var user: SBAUser = {
```

Comment:
This turns out to be a lot less trivial than it looks at first glance, since SBAUserProfileItem depends on user implementing SBANameDataSource. MockUser implements SBAUserWrapper but not SBANameDataSource, and even if it did, since it's defined in ObjC it wouldn't have access to the givenName and fullName methods which are defined only in a protocol extension.

Having the tests run against the real SBAUser shared instance will let us know if any changes to that object would break this code, and I think it's stable enough at this point that it doesn't necessarily make these tests any more fragile than they would be with a mock object.